app/views/layouts/tpi/_header.html.erb

```diff
@@ -7,23 +7,25 @@
     },
     {
       entry: 'Publications and news',
-      path: '/tpi/publications',
+      path: url_for(controller: 'publications', only_path: true)
     },
     {
       entry: 'About',
       content: @about_pages,
+      path: url_for(controller: 'home', action: 'about', only_path: true)
     },
     {
       entry: 'Newsletter',
-      path: tpi_newsletter_path,
+      path: tpi_newsletter_path
```

Comment:
Instead of checking on frontend if path contains another, you can rewrite this to
```
{
      entry: 'Newsletter',
      path: tpi_newsletter_path,	  
      active: is_active_link?(tpi_newsletter_path)
}
```

Reply:
that will also work with nested rails routes with inclusive option (but we don't have those I guess)

Reply:
I also think that the same, checking if @tpi_tool_pages, and @about_pages, could be done with that `is_active_link?` helper too.

Reply:
I can help with that, but tomorrow my time will be very sparse, only in the evening (maybe)

Reply:
Wouldn't be this little helper be enough?
https://apidock.com/rails/ActionView/Helpers/UrlHelper/current_page%3F

Reply:
hmm, maybe this would be better :+1: